### ISSUE TITLE
Whether this work is already available for tf 2.3 ?

### ISSUE THREAD

**Issue by kaiyu-tang** [ostamand/tensorflow-tabnet#1]

**Comment by ostamand** [comment]
Yes it is available for TF 2.3. 

You can simply install using:

```bash
pip install git+https://github.com/ostamand/tensorflow-tabnet.git
```

and import the classifier as follow:

```
from tabnet.models.classify import TabNetClassifier
```

Still under development, but all the important stuff is there. It gets really good results & stable training.

For example, below is the validation accuracy for the covertype dataset for only 1000 epochs (much less than in the original paper, test accuracy is 0.963)

![image](https://user-images.githubusercontent.com/1151136/91064579-6a3b9e00-e5fd-11ea-8f50-b9afc07b0a85.png)
